dan-stineback/server.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const mongoose = require('mongoose');
+const morgan = require('morgan');
+const errorHandler = require(__dirname + '/lib/error_handling');
+
+const dbPort = process.env.MONGOLAB_URI || 'mongodb://localhost/dev_db';
+mongoose.connect('mongodb://localhost/dev_db');
```

Comment:
good idea to use mongoose.connect(dbPort).  I noticed you did that in the test and what it does for you is allow you to use environment variable to specify mongo connection string, but if there is none you have a default for local development.  You wouldn't have to change your code when deploying to cloud if you had the ability to rely on local variable.